src/main/kotlin/generate/GraphRelations.kt

```diff
@@ -81,4 +81,27 @@ object GraphRelations {
 
         return res
     }
+
+    /**
+     * @return The graph that is the disjoint union of [g1] and [g2].
+     */
+    fun <V1, V2> disjointUnion(g1: SimpleGraph<V1>, g2: SimpleGraph<V2>): SimpleGraph<Int> {
```

Comment:
At first I thought that g1 and g2 needed to have disjoint vertex-sets.
Maybe do the naming & documentation clearer that it takes two (potentially overlapping) graphs and just treats them like they were disjoint.